### PR TITLE
CONTRAST-35862/UX-199 New Vulnerability Management page

### PR DESCRIPTION
**Pull request by gdvdsn-contrast**
I am uploading the draft of the new Vulnerability Management page. This page replaces https://docs.contrastsecurity.com/admin-policymgmt.html#remediate and includes the Vuln Behavior section from https://docs.contrastsecurity.com/admin-orgsettings.html#app-defaults.

I pulled this draft together from the gdoc draft. I will still need to do some style editing. Otherwise, I'll need help with making sure the content is correct.

### CODE REVIEW

File: content/admin/policy_mgmt/VulnerabilityManagement.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+See [Manage Vulnerabilities](user-vulns.html#manage-vuln) for more information about Pending states and workflows.
+
+## Vulnerability Policy
+
+Create policies for vulnerabilities in your organization, and Contrast will clean up your view of security risks. As Contrast assesses your applications, it can either flag vulnerabilities that are in violation of the policy or automatically mark them Remediated - Auto-Verified. 
+
+### How It Works 
+
+
```
Comment (dpcsgh) [inline content/admin/policy_mgmt/VulnerabilityManagement.md]: remove extra line break

File: content/admin/policy_mgmt/VulnerabilityManagement.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+Administrators can define requirements for vulnerability remediation based on any vulnerability rule, severity, application(s) and route which should comply. Contrast then marks any vulnerabilities that are in violation or Remediated - Auto-Verified based on this policy with a notification in the interface. Administrators are also notified of violations in the product and by email.
```
Comment (dpcsgh) [inline content/admin/policy_mgmt/VulnerabilityManagement.md]: Consider bolding  vulnerability rule, severity, application(s) and route. It's done in the Add policies section below where they are referred to as entry fields.

File: content/admin/policy_mgmt/VulnerabilityManagement.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+### Set Up Policies
+
+Go to the **User menu > Policy Management > Vulnerability Management tab** to create new policies and view existing ones in the tabbed grid. Auto-Verification policies will automatically change the status of vulnerability to Remediated - Auto-Verified. Violation policies will mark a vulnerability as being in violation of a policy. Either can be created to be triggered based on time or route.
```
Comment (dpcsgh) [inline content/admin/policy_mgmt/VulnerabilityManagement.md]: Consider styling where ever we refer to Remediated - Auto-Verified as a specific status, as something to stand out. I noticed we don't in the above section about Vuln behavior with regards to the pending status, but might be nice for quick scanning.

File: content/admin/policy_mgmt/VulnerabilityManagement.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+### Edit Policies 
+
+Click the name of an existing policy in the grid to see its detail view and update it. Click the trashcan icon in the grid to permanently delete the policy. Once deleted, Contrast won't notify you of vulnerabilities that were within the scope of this policy. You can also use the toggles in the grid to enable or disable individual policies, if you’d like to re-use the policy again later. 
+
+## Policy Violations and Auto-Verifications 
+
+If a Violation policy has been triggered, Contrast adds a warning note to the Vulnerabilities thermometer in the dashboard and flags the associated vulnerabilities in the Vulnerabilities grid. Hover over the warning icon for each vulnerability in the grid, or go to the vulnerability details page for more information about the violation.
+
+If an Auto-Verification policy has been triggered, its status is updated to Remediated - Auto-verified in the Vulnerabilities grids. Hover over the status text, or go the auto-generated comment in the vulnerability's **Discussion** tab for more information about the applicable policy.
```
Comment (dpcsgh) [inline content/admin/policy_mgmt/VulnerabilityManagement.md]: Last I recall, there was some issues around whether we'd be able to deliver the discussion comments for Q4. We'll have to see where that ends up. Just something to be aware of for now though.

Comment (gdvdsn-contrast) [reply content/admin/policy_mgmt/VulnerabilityManagement.md]: Hi @dpcsgh, thanks so much for your comments. So the Discussion piece in the UI may or may not make it? I read a little bit about it, but I'm still lacking some context (just from not being around long enough). Also, just to clarify, this goes out with Dec release? Also still learning release schedules. :) Thanks!

Comment (dpcsgh) [reply content/admin/policy_mgmt/VulnerabilityManagement.md]: @gdvdsn-contrast I followed up, and it seems it's been completed, but is just on an un-merged branch presently. So, it should be good to include this then.

**Comment by gdvdsn-contrast** [comment]
The new Vulnerability Management page is up here: https://docs.contrastsecurity.com/admin-policymgmt.html#vuln-mgmt. The Remediation Policy page has been removed. 

Also, this page has been updated to reflect changes to the Behavior section: https://docs.contrastsecurity.com/admin-orgsettings.html#app-defaults.

**Pull request merged**